build-support/rust-target-prefix.txt

```diff
@@ -24,7 +24,7 @@
 // Default isn't as big a deal as people seem to think it is.
 #![allow(
   clippy::new_without_default,
```

Comment:
These two are now redundant I think? Here and in the other files.

Reply:
Done